src/minimega/container.go

```diff
@@ -574,26 +574,14 @@ func NewContainer(name string) *ContainerVM {
 }
 
 func (vm *ContainerVM) Launch(ack chan int) error {
-	go vm.launch(ack)
+	vm.asyncLaunch(ack)
```

Comment:
Leaving one layer of indirection -- `*ContainerVM.launch()` is called via `*ContainerVM.Launch` and `*ContainerVM.Start` so it's a bit cleaner to have the lock acquired outside of `*ContainerVM.launch()`.